### DISCUSSION TITLE
v2/copy-code/zh/migration

### DISCUSSION THREAD

**Discussion by giscus** [vuepress-theme-hope/giscus-discussions#370] [Announcements]
# v2/copy-code/zh/migration

从 @mr-hope/vuepress-plugin-copy-code 重命名为 vuepress-plugin-copy-code2。 ⚠ 添加 pure 选项 !新增 (https://img.shields.io/badge/-New-brightgreen); 支持启用纯净模式，不再有花哨的复制按钮和成功提示。 新增 delay 选项 !新增...

https://vuepress-theme-hope.github.io/v2/copy-code/zh/migration.html

**Comment by jiyunqq** [comment]
hjkh
